KIPs/kip-34.md

```diff
@@ -410,7 +414,8 @@ Each keyring class uses the `PrivateKey` class, which has one private key as a m
 | Method | Description |
 | ----------- | ----------- |
 | getPublicKey(): List&#60;List&#60;String&#62;&#62; | Returns the public key strings for all roles. |
-| getKeyByRole(role): List&#60;PrivateKey&#62; | Returns the private keys for all roles. |
+| getPublicKey(compressed: Boolean): String | Returns the compressed public key strings for all roles if compressed is true. It returns the uncompressed public key strings otherwise. |
```

Comment:
```suggestion
| getPublicKey(compressed: Boolean): List&#60;List&#60;String&#62;&#62; | Returns the compressed public key strings for all roles if compressed is true. It returns the uncompressed public key strings otherwise. |
```